algorithms/css/algorithms.css

```diff
@@ -1,3 +1,35 @@
 td, th {
 	text-align: center !important;
 }
+
+.reveal pre {
+    display: block;
+    position: relative;
+    width: 90%;
+    margin: 15px auto;
+    text-align: left;
+    font-size: 0.55em;
+    font-family: monospace;
+    line-height: 1.2em;
+    word-wrap: break-word;
+    box-shadow: 0px 0px 6px rgba(0, 0, 0, 0.3);
+}
+
+.reveal pre code {
+	padding: 5px;
+	overflow: auto;
+	max-height: 400px;
+	word-wrap: normal;
+  text-align: left;
```

Comment:
@sara-gudeman super small but this block has weird whitespace - can you fix please?

Reply:
Will do!